ci-operator/config/stackrox/stackrox/stackrox-stackrox-master__ocp-4-15.yaml

```diff
@@ -0,0 +1,154 @@
+base_images:
+  ocp-4:
+    name: automation-flavors
+    namespace: stackrox
+    tag: openshift-4-0.9.1
+  ubi-minimal:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+build_root:
+  image_stream_tag:
+    name: apollo-ci
+    namespace: stackrox
+    tag: stackrox-test-0.3.61
+resources:
+  '*':
+    requests:
+      cpu: 2000m
+      memory: 4000Mi
+test_binary_build_commands: .openshift-ci/dispatch.sh test-binary-build-commands
+tests:
+- always_run: false
+  as: qa-e2e-tests
+  steps:
+    env:
+      COLLECTION_METHOD: core_bpf
+      OCP_VERSION: ocp-dev-preview/candidate-4.15
+      TEST_SUITE: ocp-qa-e2e-tests
+    workflow: stackrox-automation-flavors-ocp-4-e2e
+  timeout: 5h0m0s
+- always_run: false
+  as: ebpf-qa-e2e-tests
+  optional: true
+  steps:
+    env:
+      COLLECTION_METHOD: ebpf
+      OCP_VERSION: ocp-dev-preview/candidate-4.15
+      TEST_SUITE: ocp-qa-e2e-tests
+    workflow: stackrox-automation-flavors-ocp-4-e2e
+  timeout: 5h0m0s
+- always_run: false
+  as: operator-e2e-tests
+  optional: true
+  steps:
+    env:
+      OCP_VERSION: ocp-dev-preview/candidate-4.15
+      TEST_SUITE: ocp-operator-e2e-tests
+    workflow: stackrox-automation-flavors-ocp-4-e2e
+  timeout: 5h0m0s
+- always_run: false
+  as: sensor-integration-tests
+  optional: true
+  steps:
+    env:
+      COLLECTION_METHOD: core_bpf
+      OCP_VERSION: ocp-dev-preview/candidate-4.15
+      TEST_SUITE: ocp-sensor-integration-tests
+    workflow: stackrox-automation-flavors-ocp-4-e2e
+  timeout: 5h0m0s
+- always_run: false
+  as: ui-e2e-tests
+  optional: true
+  steps:
+    env:
+      COLLECTION_METHOD: core_bpf
+      OCP_VERSION: ocp-dev-preview/candidate-4.15
+      TEST_SUITE: ocp-ui-e2e-tests
+    workflow: stackrox-automation-flavors-ocp-4-e2e
+  timeout: 5h0m0s
+- always_run: false
+  as: fips-qa-e2e-tests
+  optional: true
+  steps:
+    env:
+      COLLECTION_METHOD: core_bpf
+      FIPS_ENABLED: "true"
+      OCP_VERSION: ocp-dev-preview/candidate-4.15
+      TEST_SUITE: ocp-qa-e2e-tests
+    workflow: stackrox-automation-flavors-ocp-4-e2e
+  timeout: 5h0m0s
+- always_run: false
+  as: compliance-e2e-tests
+  optional: true
+  steps:
+    env:
+      COLLECTION_METHOD: core_bpf
+      OCP_VERSION: ocp-dev-preview/candidate-4.15
+      TEST_SUITE: ocp-compliance-e2e-tests
+    workflow: stackrox-automation-flavors-ocp-4-e2e
+  timeout: 5h0m0s
+- always_run: false
+  as: sensor-profiling
+  optional: true
+  steps:
+    env:
+      COLLECTION_METHOD: core_bpf
+      OCP_VERSION: ocp-dev-preview/candidate-4.15
+      TEST_SUITE: ocp-sensor-profiling
+    workflow: stackrox-automation-flavors-ocp-4-e2e
+  timeout: 2h0m0s
+- always_run: false
+  as: merge-qa-e2e-tests
```

Comment:
I'd delete the merge/postsubmit jobs. I'm pretty sure they do not respect `always_run: false`. And you won't know until after this PR is merged. https://docs.ci.openshift.org/docs/how-tos/contributing-openshift-release/#tolerated-changes-to-generated-jobs